Cargo.toml

```diff
@@ -20,18 +20,18 @@ hyper = { version = "0.14", features = ["client"] }
 tower-service = "0.3"
 http = "0.2"
 futures-util = { version = "0.3", default-features = false }
-bytes = "1.0"
-hyper-tls = { version = "0.5.0", optional = true }
-tokio-native-tls = { version = "0.3.0", optional = true }
+bytes = "1"
+hyper-tls = { version = "0.5", optional = true }
+tokio-native-tls = { version = "0.3", optional = true }
 native-tls = { version = "0.2", optional = true }
 openssl = { version = "0.10", optional = true }
 tokio-openssl = { version = "0.6", optional = true }
-tokio-rustls = { version = "0.22", optional = true }
-hyper-rustls = { version = "0.22", optional = true }
+tokio-rustls = { version = "0.24", optional = true }
+hyper-rustls = { version = "0.24", optional = true }
 
-webpki = { version = "0.21", optional = true }
-rustls-native-certs = { version = "0.5.0", optional = true }
-webpki-roots = { version = "0.21.0", optional = true }
```

Comment:
Hope to see this MR merged soon. Currently facing https://rustsec.org/advisories/RUSTSEC-2023-0052 from this.